Misc/NEWS.d/next/Library/2024-04-30-22-18-30.gh-issue-118455._XnYvo.rst

```diff
@@ -0,0 +1 @@
+Correct default value of mangle_from\_ in email.policy.Policy docstring
```

Comment:
We don't need NEWS for a doc-only change.

Reply:
Agree, but the CI fails without that and I don't have permission to add labels 🤷